CMakeLists.txt

```diff
@@ -358,21 +358,24 @@ ENDIF()
 # Find the libdap package so that Aspect can connect to the OPeNDAP servers
 # Author: Kodi Neumiller
 # Added: 10/16/18
-LIST(APPEND CMAKE_MODULE_PATH "${CMAKE_MODULE_PATH}/cmake/modules")
-MESSAGE(STATUS "cmake module path = ${CMAKE_MODULE_PATH}")
-FIND_PACKAGE(LIBDAP)
-IF(${LIBDAP_FOUND})
-  FIND_PACKAGE(LibXml2)
-  FIND_PACKAGE(CURL)
-  INCLUDE_DIRECTORIES(${LIBDAP_INCLUDE_DIR} ${LIBXML2_INCLUDE_DIR} ${CURL_INCLUDE_DIR})
-  TARGET_LINK_LIBRARIES(${TARGET} ${LIBDAP_LIBRARIES} ${LIBXML2_LIBRARIES} ${CURL_LIBRARIES})
-    MESSAGE(STATUS "libdap found at ${LIBDAP_LIBRARIES}")
-  
-ELSE()
-  MESSAGE(STATUS "libdap package not found")
+IF(LIBDAP_ON) 
+	LIST(APPEND CMAKE_MODULE_PATH "${CMAKE_MODULE_PATH}/cmake/modules")
+	MESSAGE(STATUS "cmake module path = ${CMAKE_MODULE_PATH}")
+	FIND_PACKAGE(LIBDAP)
+	IF(${LIBDAP_FOUND})
+	  FIND_PACKAGE(LibXml2)
+	  FIND_PACKAGE(CURL)
+	  INCLUDE_DIRECTORIES(${LIBDAP_INCLUDE_DIR} ${LIBXML2_INCLUDE_DIR} ${CURL_INCLUDE_DIR})
+	  TARGET_LINK_LIBRARIES(${TARGET} ${LIBDAP_LIBRARIES} ${LIBXML2_LIBRARIES} ${CURL_LIBRARIES})
+	    MESSAGE(STATUS "libdap found at ${LIBDAP_LIBRARIES}")
+	  
+	ELSE()
+	  MESSAGE(STATUS "libdap package not found")
+	ENDIF()
+	
+	ADD_DEFINITIONS(-DHAVE_LIBDAP=TRUE)
```

Comment:
Should this be inside the 'then' clause of the 'IF (${LIBDAP_FOUND})...' statement?

Reply:
Yes, I don't know why/how it got put outside the ENDIF()